frontend/src/pages/class/Insights.jsx

```diff
@@ -6,6 +6,7 @@ import { DataGrid } from '@material-ui/data-grid';
 import PaperWithHeader from '../_shared/PaperWithHeader';
 import ProgressBar from '../_shared/ProgressBar';
 import PageContainer from '../_shared/PageContainer';
+import { useHistory } from 'react-router-dom';
```

Comment:
Het is altijd een goeie gewoonte om imports / requires die van npm komen *boven* de lokale imports te zetten